stages/org.osbuild.chown

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python3
+"""
+Change file owner and group
+
+Change the file user and/or group ownership of one or more files or directories inside the tree.
+"""
+
+import subprocess
+import sys
+import os
+
+import osbuild.api
+
+SCHEMA = """
+"additionalProperties": false,
+"required": ["paths"],
+"properties": {
+  "paths": {
+    "type": "array",
+    "description": "Paths to operate on for changing ownership",
+    "items": {
+      "type": "string"
+    }
+  },
+  "user": {
+    "type": "string"
+  },
+  "group": {
+    "type": "string"
+  },
+  "recursive": {
+    "type": "boolean"
+  }
+}
+"""
+
+def main(tree, options):
+    paths = options["paths"]
+    user = options.get("user", "")
+    group = options.get("group", "")
```

Comment:
So, here is an interesting question: do we only allow numeric values? (and thus change the type to `int`) or do we want to resolve `user` and `group` via the respective files in the tree.
`chown` internally seems to use `getpwnam` via `parse_with_separator` (gnulib), which goes down the `glibc` name services route. I guess running `chown` inside a `chroot` would work too, but seems iffy too. Python, sadly enough does not have `fgetpwent`, but a bare bones version was implemented by Martin for PR #620

Reply:
I hate the nss plugins in a chroot, but we may want to consider what rpm does and stay compatible (there are three potential interpretations: use the host, the buildroot or the target, host is obviously not an option, but I don't recall which of the others rpm uses)? Unless we are able to just stick to numeric values.